pkgs/development/python-modules/safeio/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, fetchpatch }:
```

One input per line help in the future to keep the diffs better readable.

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, fetchpatch
}:
```